libkbfs/file_data_test.go

```diff
@@ -405,6 +411,11 @@ func testFileDataLevelExistingBlocks(t *testing.T, fd *fileData,
 		prevChildren = level
 		numLevels++
 	}
+
+	if numLevels > 1 {
+		fd.file.path[len(fd.file.path)-1].DirectType = IndirectBlock
```

Comment:
So you're setting the last pointer in the path to `IndirectBlock`. Shouldn't the last one be `DirectBlock`?